extension/json/json_serializer.cpp

```diff
@@ -216,4 +223,13 @@ void JsonSerializer::WriteDataPtr(const_data_ptr_t ptr, idx_t count) {
 	PushValue(val);
 }
 
```

Comment:
Add a function to write a timestamp value to the JSON document, formatted according to the specified timestamp format.